apps/web/src/app/(dashboard)/dashboard/layout.tsx

```diff
@@ -1,12 +1,14 @@
-import { LayoutDashboard } from '@/components/LayoutDashboard'
+import { FooterMain } from '@/components/Footer'
 import { Toaster } from '@/components/ui/toaster'
 
-
 export default async function Layout({ children }) {
   return (
-    <LayoutDashboard>
-      {children}
-      <Toaster />
-    </LayoutDashboard>
+    <div className="h-full w-full">
+      <main className="w-full h-full flex flex-col">
```

Comment:
Why didn't you update LayoutDashboard instead?

Reply:
I deleted the component and moved the layout directly to this `layout.tsx` file. The `LayoutDashboard` wasn't used anywhere but here and there wasn't much logic in it.